packages/sdk/src/index.ts

```diff
@@ -125,7 +124,7 @@ export class Gomu {
   async makeOrder({
     marketplaces,
     ...params
-  }: MakeOrderParams): Promise<MakeOrderResponse[]> {
+  }: MakeOrderParams): Promise<Order[]> {
```

Comment:
Addendum for clarity:
Structure of `marketplaceName` + `data` + `error` is ok
Just that something like `OrderResponse` is more appropriate than `Order` since "orders" technically shouldn't contain errors